.github/pull_request_template.md

```diff
@@ -1,43 +1,59 @@
 <!--
-Thank you for contributing to TiDB Operator! Please read TiDB Operator's [CONTRIBUTING](https://github.com/pingcap/tidb-operator/blob/master/CONTRIBUTING.md) document **BEFORE** filing this PR.
+Thank you for contributing to TiDB Operator!
+Please complete the following template before creating a PR :D
 -->
 
-### What problem does this PR solve? <!--add and issue link with summary if exists-->
+### What problem does this PR solve?
+<!--
+Please describe the problem AS DETAILED AS POSSIBLE.
+Add and ISSUE LINK WITH SUMMARY if exists.
+
+For example:
+
+    Fix the bug that syncing `Backup` CR will crash tidb-controller-manager pod when client TLS feature is enabled.
+
+    Closes #xxx (issue number)
+-->
 
 ### What is changed and how does it work?
+<!--
+Please describe the design that your implementation follows AS DETAILED AS POSSIBLE.
 
-### Check List <!--REMOVE the items that are not applicable-->
+For example:
 
-Tests <!-- At least one of them must be included. -->
+    The root cause is a nil pointer dereferencing. Add a `ptr != nil` check before access members of `ptr` to prevent the crash.
+-->
+
+### Code changes
 
-- Unit test
-- E2E test
-- Stability test
-- Manual test (add detailed scripts or steps below)
-- No code
+- [ ] Has Go code change
+- [ ] Has CI related scripts change
 
-Code changes
+### Tests
+<!-- AT LEAST ONE test must be included. -->
 
-- Has Go code change
-- Has CI related scripts change
-- Has Terraform scripts change
+- [ ] Unit test <!-- If you added any unit test cases, check this box -->
+- [ ] E2E test <!-- If you added any e2e test cases, check this box -->
+- [ ] Manual test <!-- If this PR needs manual test, check this box, and add detailed manual test scripts or steps below, so that ANYONE CAN REPRODUCE IT. Ref: https://github.com/pingcap/tidb-operator/pull/3517 -->
+- [ ] No code <!-- If this PR contains no code changes, check this box -->
 
-Side effects
+### Side effects
 
-- Breaking backward compatibility
+- [ ] Breaking backward compatibility <!-- If this PR breaks things deployed with previous TiDB Operator versions, check this box -->
+- [ ] Other side effects: <!-- Any other side effects, such as requiring additional storage / consumes substantial memory / potential reconcilation latency -->
 
-Related changes
+### Related changes
 
-- Need to cherry-pick to the release branch
-- Need to update the documentation
+- [ ] Need to cherry-pick to the release branch <!-- If this PR should also appear in the current release branch, check this box -->
```

Comment:
we have two release branches, how about given two checkboxes for both versions?

Reply:
Then this template must be changed every time we add a new release branch
Is there a better way of doing this? 🤔

Reply:
I prefer to change this template for every release (we know the PR should be cherry-picked to which branch(es)). but still waiting for opinions from other reviewers.

Reply:
I don't think change template with code release is a good idea.
Maybe we could make the committer write the release number?
WDYT @lonng @DanielZhangQD

Reply:
I think it's enough to let the author of PR marks whether this PR need to be cherry-picked, and the reviewer add some labels for dedicate release branches.